NewscronJSP/src/main/java/ch/newscron/newscronjsp/ShortUrlStatistics.java

```diff
@@ -40,41 +34,40 @@ public String showStatisticsTable(String custID) throws IOException {
         return toReturn;
     }
     
-    public String showUserTable() throws IOException, SQLException {
-        ResultSet rs = UserRegistration.selectAllUsers();
+    public String showUserTable() throws IOException {
+        List<User> userList = UserRegistration.selectAllUsers();
         
         String toReturn = "<h3> All Users </h3>" 
                         + "<table border='1' class=\"center\"> "
                         + "<tr> <td colspan='4'> User Table </td> <td class='red' colspan='2'> ShortURL Table </td> </tr>"
-                        + "<tr> <td> Name </td> <td> Last Name </td>"
-                        + "<td> email </td> <td> campaignId </td> <td class='red'> invitedBy </td> <td class='red'> invitationURL </td> </tr>";
+                        + "<tr> <th> Name </th> <th> Last Name </th>"
+                        + "<th> email </th> <th> campaignId </th> <th class='red'> invitedBy </th> <th class='red'> invitationURL </th> </tr>";
         
         String name;
         String lastName;
         String email;
-        String campaignId;
+        long campaignId;
         String invitedBy;
         String invitationURL;
         
         //LOOP OVER RESULT SET
-        while (rs.next()) {
-                name = rs.getString("name");
-                lastName = rs.getString("lastName");
-                email = rs.getString("email");
-                campaignId = rs.getString("campaignId");
-                invitedBy = rs.getString("custID");
-                invitationURL = rs.getString("shortURL");
-
-        toReturn += "<tr> <td>" + name + "</td>";
-        toReturn += "<td>" + lastName + "</td>";
-        toReturn += "<td>" + email + "</td>";
-        toReturn += "<td>" + campaignId + "</td>";
-        toReturn += "<td class='red'>" + invitedBy + "</td>";
-        toReturn += "<td class='red'>" + invitationURL + "</td>";
+        for (User userList1 : userList) {
+            name = userList1.getFirstName();
+            lastName = userList1.getLastName();
+            email = userList1.getEmail();
+            campaignId = userList1.getCampaignId();
+            invitedBy = userList1.getInvitedBy();
+            invitationURL = userList1.getInvitationURL();
+            
+            toReturn += "<tr> <td>" + name + "</td>";
```

Comment:
do you need distinguish variable for that? Just enough to use userList1.getFirstName().

BTW.  userList1 is bad name